test/Microsoft.IdentityModel.Logging.Tests/project.json

```diff
@@ -1,14 +1,28 @@
 ﻿{
   "version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*"
+    "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*",
+    "xunit": "2.1.0"
   },
   "testRunner": "xunit",
   "frameworks": {
     "netstandardapp1.5": {
-      "imports": [ "dnxcore50", "portable-net451+win8" ]
+      "dependencies": {
+        "dotnet-test-xunit": "1.0.0-dev-*",
+        "NETStandard.Library": "1.5.0-*",
+        "System.Diagnostics.Process": "4.1.0-*"
+      },
+      "imports": [
+        "dnxcore50",
+        "portable-net451+win8"
+      ]
     },
-    "dnx451": {
+    "net451": {
+      "frameworkAssemblies": {
```

Comment:
thanks, I will see what happens. seems to build fine without them. I will remove them.